dist/components/badge/_badge.scss

```diff
@@ -0,0 +1,30 @@
+// Component Name
+// ===
+//
+// A badge that shows the quantity of items in the cart, often used with a cart icon in the header.
+
+
+// Component Variables
+// ---
+//
+
+$badge__background-color: red !default;
+$badge__text-color: white !default;
+$badge__border-radius: 50% !default;
+$badge__size: 15px !default;
+$badge__font-size: 10px !default;
+
+// Component Base
+// ---
+
+.c-badge {
+    width: $badge__size;
```

Comment:
Should width perhaps be `min-width`? I suppose just in case we want to allow more than two digits to work inside the badge?